frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/review-tab/networking-review.tsx

```diff
@@ -1,34 +1,48 @@
 import * as React from 'react';
-import * as _ from 'lodash';
 import { connect } from 'react-redux';
 import { VMWizardNetwork } from '../../types';
 import { getNetworks } from '../../selectors/selectors';
-import { ReviewList } from './review-list';
 import { NetworkInterfaceWrapper } from '../../../../k8s/wrapper/vm/network-interface-wrapper';
 import { NetworkWrapper } from '../../../../k8s/wrapper/vm/network-wrapper';
+import { Table, TableBody, TableHeader, TableVariant } from '@patternfly/react-table';
 
 const NetworkingReviewConnected: React.FC<NetworkingTabComponentProps> = ({
   networks,
   className,
-}) => (
-  <ReviewList
-    title="Network Interfaces"
-    className={className}
-    items={networks.map(({ id, networkInterface, network }) => {
-      const networkInterfaceWrapper = new NetworkInterfaceWrapper(networkInterface);
-      const networkWrapper = new NetworkWrapper(network);
-      return {
-        id,
-        value: _.compact([
-          networkInterfaceWrapper.getName(),
-          networkInterfaceWrapper.getReadableModel(),
-          networkWrapper.getReadableName(),
-          networkInterfaceWrapper.getMACAddress(),
-        ]).join(' - '),
-      };
-    })}
-  />
-);
+}) => {
+  const headers = [
+    { title: 'Name' },
+    { title: 'Model' },
+    { title: 'MAC Address' },
+    { title: 'Network' },
+  ];
+
+  const rows = networks.map(({ networkInterface, network }) => {
+    const networkInterfaceWrapper = new NetworkInterfaceWrapper(networkInterface);
+    const networkWrapper = new NetworkWrapper(network);
+
+    return [
+      networkInterfaceWrapper.getName(),
+      networkInterfaceWrapper.getReadableModel(),
+      networkInterfaceWrapper.getMACAddress(),
+      networkWrapper.getReadableName(),
+    ];
+  });
+
+  return (
+    <Table
+      aria-label="Network Interfaces"
+      variant={TableVariant.compact}
+      cells={headers}
+      rows={rows}
+      className={className}
+      gridBreakPoint="grid-2xl"
```

Comment:
can you change this to `grid-xl` and also add it to the storage table?

Reply:
Done!